nanoFramework.M2Mqtt.sln

```diff
@@ -1,20 +1,25 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31410.357
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32901.215
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "nanoFramework.M2Mqtt", "M2Mqtt\nanoFramework.M2Mqtt.nfproj", "{35948E70-4CE5-45B4-B305-B66B20AB67AD}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{06FC0A0E-446E-4173-88DA-535ECD697BE2}"
 	ProjectSection(SolutionItems) = preProject
-		M2Mqtt\nanoFramework.M2Mqtt.nuspec = M2Mqtt\nanoFramework.M2Mqtt.nuspec
+		.runsettings = .runsettings
```

Comment:
I don't see anymore the runsettings file. Seems that it has been deleted

Reply:
It is in the root directory...